### PR TITLE
Fix bug #1923

### PR DESCRIPTION
**Pull request by dreamsxin**

### CODE REVIEW

File: phalcon/mvc/model/query/builder.zep [hunk 1]
```diff
@@ -1370,6 +1370,12 @@ class Builder implements BuilderInterface, InjectionAwareInterface
 			query->setBindTypes(bindTypes);
 		}
 
+		if typeof this->_sharedLock === "boolean" {
+			if this->_sharedLock {
+				query->setSharedLock(true);
+			}
```
Comment (sergeyklay) [inline phalcon/mvc/model/query/builder.zep]: Why not:
```zephir
if typeof this->_sharedLock === "boolean" {
    query->setSharedLock(this->_sharedLock);
}
```

Comment (dreamsxin) [inline phalcon/mvc/model/query/builder.zep]: Because in the query default is false,  But you are right.

**Comment by sergeyklay** [comment]
:+1:

**Comment by andresgutierrez** [comment]
Thanks

**Pull request merged**